site/content/installation-upgrade/container-environments/docker-images.md

```diff
@@ -199,6 +206,11 @@ docker run --name nginx-agent -d \
   nginx-agent
 ```
 
+To ensure that the REST Interface is correctly configured, you can curl the following endpoint from your terminal:
+```shell
+curl 0.0.0.0:8038/nginx/
+```
+
```

Comment:
Should we also provide an example output or what to expect from this command to confirm the test?